### PR TITLE
Add checking error for disruption e2e test

### PR DESCRIPTION
**Pull request by k-toyoda-pi**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
There is no error checking after getting the status in test/e2e/disruption.go.
Therefore, this adds the check.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
```docs

```

### CODE REVIEW

File: test/e2e/apps/disruption.go [hunk 1]
```diff
@@ -607,6 +607,7 @@ func waitForPdbToObserveHealthyPods(cs kubernetes.Interface, ns string, healthyC
 func getPDBStatusOrDie(dc dynamic.Interface, ns string, name string) *policyv1beta1.PodDisruptionBudget {
 	pdbStatusResource := policyv1beta1.SchemeGroupVersion.WithResource("poddisruptionbudgets")
 	unstruct, err := dc.Resource(pdbStatusResource).Namespace(ns).Get(context.TODO(), name, metav1.GetOptions{}, "status")
+	framework.ExpectNoError(err)
```
Comment (tanjunchen) [inline test/e2e/apps/disruption.go:610]: Thanks.
/lgtm

**Review by oomichi — COMMENTED** [review]
Thanks for doing this.

/lgtm
/approve

**Comment by k-toyoda-pi** [comment]
@tanjunchen @oomichi thank you for reviewing!

**Pull request merged**